src/Parser/Hunter/Survival/Modules/Talents/WayOfTheMokNathal.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'Parser/Core/Analyzer';
+import StatisticBox from 'Main/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import STATISTIC_ORDER from 'Main/STATISTIC_ORDER';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+
+const MAX_STACKS = 4;
+
+class WayOfTheMokNathal extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  _currentStacks = 0;
+  fourStackUptime = 0;
+  fourStackStart = 0;
```

Comment:
should this be a private variable? don't see a use for it outside of this module.

Reply:
Have we ever done that in any modules? 😮

Reply:
"private" meaning "stick a _ in front so other people know it may break underneath them"

Reply:
Done 👌